js/flightlog_parser.js

```diff
@@ -582,23 +582,20 @@ var FlightLogParser = function(logData) {
                 // Extract the firmware revision in case of Betaflight/Raceflight/Cleanfligh 2.x/Other
                 var matches = fieldValue.match(/(.*flight).* (\d+)\.(\d+)(\.(\d+))*/i);
                 if(matches!=null) {
-                
-                    // The that.sysConfig.firmwareType usually has been detected before that. Default value.
-                    if (typeof that.sysConfig.firmwareType === 'undefined' || !that.sysConfig.firmwareType) {
-                	    that.sysConfig.firmwareType = FIRMWARE_TYPE_BETAFLIGHT;                    
```

Comment:
This cant be retained as is, this is what disables the betaflight detection - in the case of betaflight other checks pre-set that.sysConfig.firmwareType to FIRMWARE_TYPE_CLEANFLIGHT and causes betaflight to not be detected.